corehq/pillows/mappings/domain_mapping.py

```diff
@@ -211,6 +211,7 @@
                 'project_type': {'analyzer': 'comma', 'type': 'string'},
                 'published': {'type': 'boolean'},
                 'publisher': {'type': 'string'},
+                'allow_domain_requests': {'type': 'boolean'},
```

Comment:
Looks like a merge caused the index to not be updated, so this property won't be indexed. I don't actually see where you refer to this in this PR (at least from ES standpoint) so I'm not sure that it'll actually make a difference.

Should either remove this line or update index